require touches along with shaking

jhocking opened the pull request:
Ignore this change if you don't want it, but I found it to be a big improvement. Instead of only shaking on mobile, I made it require 3 or more fingers on the screen while shaking. The reason for this was that testers kept opening the console by accident.

mminer commented:
Interesting, this looks like a useful change. Accidentally opening the console is the pits.

I think it'd make sense as a configurable option; I'd hate to change the default behaviour for people that use shake-to-open. Any chance you could add something like a `shakeToOpenRequiresTouch` boolean option to the inspector?

jhocking commented:
Cool, I added that boolean option and tested to make sure it works. Incidentally, I also changed it from only making visibility true to a toggle so that the console can be closed again. Unless I was missing it, there was no way to close the console on mobile devices.

mminer commented:
Great, thanks for adding the boolean option. Everything looks A+ to me. Thanks for the PR!
